/camera_node:
  ros__parameters:
    camera_name: hik_01
    exposure_time: 2000
    gain: 5.0
